package.json

```diff
@@ -26,80 +27,71 @@
-    "@storybook/storybook-deployer": "^2.3.0",
-    "babel-cli": "^6.24.1",
-    "babel-jest": "^23.0.0",
-    "babel-loader": "^7.0.0",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-env": "^1.4.0",
+    "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/config-angular": "^7.5.0",
+    "@commitlint/prompt-cli": "^7.5.0",
+    "@edx/edx-bootstrap": "^1.0.4",
+    "@storybook/addon-a11y": "^5.0.3",
+    "@storybook/addon-actions": "^5.0.3",
+    "@storybook/addon-centered": "^5.0.3",
+    "@storybook/addon-console": "^1.1.0",
+    "@storybook/addon-info": "^5.0.3",
+    "@storybook/addon-storyshots": "^5.0.3",
+    "@storybook/react": "^5.0.3",
+    "@storybook/storybook-deployer": "^2.8.1",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.5",
+    "babel-plugin-require-context-hook": "^1.0.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-react": "^6.24.1",
-    "coveralls": "^3.0.0",
-    "css-loader": "^0.28.4",
-    "enzyme": "^3.1.1",
-    "enzyme-adapter-react-16": "^1.0.4",
-    "es-check": "^3.0.0",
-    "eslint": "^4.5.0",
-    "eslint-config-edx": "^4.0.3",
-    "eslint-import-resolver-webpack": "^0.10.0",
-    "eslint-plugin-compat": "^2.2.0",
-    "extract-text-webpack-plugin": "^3.0.1",
-    "file-loader": "^1.1.4",
-    "greenkeeper-lockfile": "^1.7.1",
-    "husky": "^1.0.0",
+    "bootstrap": "^4.3.1",
+    "clean-webpack-plugin": "^2.0.1",
+    "coveralls": "^3.0.3",
+    "css-loader": "^2.1.1",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.11.2",
+    "es-check": "^5.0.0",
+    "eslint-config-edx": "^4.0.4",
+    "greenkeeper-lockfile": "^1.15.1",
+    "husky": "^1.3.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^22.0.4",
-    "jest-cli": "^22.0.4",
+    "jest": "^24.5.0",
     "markdown-loader-jest": "^0.1.1",
-    "node-sass": "^4.9.4",
-    "postcss-scss": "^2.0.0",
-    "react-router-dom": "^4.1.1",
-    "react-test-renderer": "^16.1.0",
-    "sass-loader": "^6.0.5",
-    "semantic-release": "^15.1.7",
-    "source-map-loader": "^0.2.1",
-    "storybook-readme": "^3.3.0",
-    "style-loader": "^0.22.0",
-    "travis-deploy-once": "^5.0.0",
-    "uglifyjs-webpack-plugin": "^1.3.0",
-    "webpack": "^3.0.0",
-    "webpack-build-notifier": "^0.1.22",
-    "webpack-dev-server": "^3.0.0"
+    "mini-css-extract-plugin": "^0.5.0",
+    "node-sass": "^4.11.0",
+    "sass-loader": "^7.1.0",
+    "semantic-release": "^15.13.3",
+    "source-map-loader": "^0.2.4",
+    "storybook-readme": "^4.0.5",
+    "style-loader": "^0.23.1",
+    "travis-deploy-once": "^5.0.11",
+    "uglifyjs-webpack-plugin": "^2.1.2",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.3.0"
   },
   "jest": {
+    "testURL": "http://localhost/",
```

Comment:
this appears to be the default https://jestjs.io/docs/en/configuration.html#testurl-string

Reply:
I think I added this due to an issue with Storyshots. I can remove and see if it breaks anything.

Reply:
No problems removing it 👍